autoPyTorch/pipeline/components/setup/network_backbone/ShapedResNetBackbone.py

```diff
@@ -153,8 +154,11 @@ def get_hyperparameter_search_space(dataset_properties: Optional[Dict] = None,
 
         use_dropout = CategoricalHyperparameter(
             "use_dropout", choices=use_dropout[0], default_value=use_dropout[1])
+        use_batch_normalization = CategoricalHyperparameter(
+            "use_batch_norm", choices=use_batch_norm[0], default_value=use_batch_norm[1])
```

Comment:
I did not use the `use_batch_norm` name, because it shadows the variable from the method.

Reply:
> Comment for the function def `get_hyperparameter_search_space` in `autoPyTorch/pipeline/components/setup/network_backbone/ResNetBackbone.py`.
> 
> Tuple[Tuple[bool], bool]?
> I think if `use_batch_norm=False` then deactivate it and `use_batch_norm=True` then creating config space looks better.
> In any case, there are only six possible choices for this.
> 
> ```
> ((True, False), True)
> ((True, False), False)
> 
> no need to define config space for the candidates below, because it is a fixed parameter.
> ((True), True)
> ((True), False)
> ((False), True)
> ((False), False)
> ```
> 
> Based on this, we have four choices:
> if we fix the parameter or not and the default parameter,
> so should be `use_batch_norm` (default) should be bool and `search_for_batchnorm` (or any other name) should be another bool variable.

@nabenabe0928 I disagree, because this would be inconsistent with `use_dropout`. It can be improved in the future.